test/tests/rackhd20/test_rackhd20_api_notifications.py

```diff
@@ -0,0 +1,271 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+Author(s):
+'''
+
+from time import sleep
+import Queue
+import flogging
+import logging
+import pika
+import unittest
+import threading
+import fit_common
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+amqp_queue = Queue.Queue(maxsize=0)
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+
+
+IP = None
+AMQP_PORT = fit_common.fitports()['amqp_ssl']
+MONGO_PORT = fit_common.fitports()['mongo_port']
+HTTP_PORT = fit_common.fitports()['http']
+env_vagrant = True
+
+
+class AmqpWorker(threading.Thread):
+    def __init__(self, exchange_name, topic_routing_key, external_callback, timeout=10):
+        threading.Thread.__init__(self)
+        pika_logger = logging.getLogger('pika')
+        if fit_common.VERBOSITY >= 8:
+            pika_logger.setLevel(logging.DEBUG)
+        elif fit_common.VERBOSITY >= 4:
+            pika_logger.setLevel(logging.WARNING)
+        else:
+            pika_logger.setLevel(logging.ERROR)
+        self.connection = pika.BlockingConnection(pika.ConnectionParameters(host=fit_common.fitargs()["rackhd_host"],
+                                                                            port=AMQP_PORT))
+        self.channel = self.connection.channel()
+        result = self.channel.queue_declare(exclusive=True)
+        queue_name = result.method.queue
+        self.channel.queue_bind(exchange=exchange_name, queue=queue_name, routing_key=topic_routing_key)
+        self.channel.basic_consume(external_callback, queue=queue_name)
+        self.connection.add_timeout(timeout, self.dispose)
+
+    def dispose(self):
+        logs.debug_3('Pika connection timeout')
+        if self.connection.is_closed is False:
+            self.channel.stop_consuming()
+            logs.debug_1("Attempting to close the Pika connection")
+            self.thread_stop = True
+
+    def run(self):
+        logs.debug_7('start consuming')
+        self.channel.start_consuming()
+
+
+@attr(all=True, regression=False, smoke=True)
+class test_alert_notification(unittest.TestCase):
+    CLIENT = None
+    MONGO_PORT = None
```

Comment:
This declaration of MONGO_PORT does not serve a purpose.   There is no use of self.MONGO_PORT.  I believe you can just remove this.